{{cookiecutter._project_name}}/generate.py

```diff
@@ -43,7 +43,8 @@
     '',
     'USER extractor'
     '',
-    'COPY algorithm_rgb.py /home/extractor/'
+    'COPY algorithm_rgb.py /home/extractor/',
+    'COPY testing.py /home/extractor/'
```

Comment:
Why is this file copied?

Reply:
When I try the docker run command: 

```
cmd = 'docker run --rm --mount "src=`pwd`,target=/mnt,type=bind" ' + algorithm_rgb.ALGORITHM_NAME + ":" + algorithm_rgb.VERSION + ' --working_space "/mnt"'
for filename in os.listdir("sample_plot_images"):
    cmd += ' "/mnt/sample_plot_images/' + filename + '"'
os.system(cmd)
```

and don't include it it gives me a "no module named algorithm_rgb error"

Reply:
Sorry, the comment was for line 47 - why is testing.py included?

Reply:
Just fixed